server/manager/src/test/java/org/apache/accumulo/manager/upgrade/Upgrader11to12Test.java

```diff
@@ -395,13 +401,20 @@ public void upgradeZooKeeperTest() throws Exception {
 
     expect(zrw.exists("/problems")).andReturn(false).once();
 
-    replay(context, zk, zrw);
+    expect(zrw.putPersistentData(isA(String.class), isA(byte[].class), isA(NodeExistsPolicy.class)))
+        .andReturn(true).times(7);
+    expect(context.getPropStore()).andReturn(store).atLeastOnce();
+    expect(store.exists(isA(TablePropKey.class))).andReturn(false).atLeastOnce();
+    store.create(isA(TablePropKey.class), isA(Map.class));
+    expectLastCall().once();
```

Comment:
You could probably use anyObject() for these parameters. It would avoid the need to suppress the warning, I think. Or, you could get more specific, and use `eq()` to pass specific objects. For the Array, you could pass a more specific matcher.

At the very least, a comment to explain where the count of 7 comes from would be useful (3 for name/conf/status for 2 tables, plus 1 for the mapping update? or something like that)